src/components/clusterCategory.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import { panel, clusterCategory } from './clusterCategory.module.css'
+import OpenInNewTabSvg from '../assets/icons/openInNewTab.svg'
+import ChevronRightSvg from '../assets/icons/chevronRight.svg'
+import ClusterIconSvg from '../assets/icons/clusterIcon.svg'
+import CategoryIconSvg from '../assets/icons/categoryIcon.svg'
+
+function Panel({ icon, title, body, style }) {
+  return (
+    <div className={`${panel}`} style={style}>
+      <div>
+        {icon}
+        {title}
+      </div>
+      {body}
```

Comment:
Can you add in the text here along with the variable cluster/category depending on data?